src/utilities.ts

```diff
@@ -110,47 +112,29 @@ function pyCrc32IsAvailable(): boolean {
   return lastOutputLine == "AVAILABLE";
 }
 
-function CRCutilityIsAvailable() {
-  // check if the crc32 utility has been added to the the VectorCAST installation
 
-  // check the value of the option, I know this is extra work if
-  // the vcast installation is on the path but this is done once and it make the code cleaner
-  let settings = vscode.workspace.getConfiguration("vectorcastTestExplorer");
-  const installationLocation = settings.get("vcastInstallationLocation", "");
+function CRCutilityIsAvailable(vcastInstallationPath:string) {
+
+  // check if the crc32 utility has been added to the the VectorCAST installation
 
   let returnValue: string | undefined = undefined;
-  if (process.platform == "linux") {
-    if (hasbin.sync(linuxCRC32)) {
-      returnValue = linuxCRC32;
-    } else if (installationLocation.length > 0) {
-      let candidatePath = path.join(installationLocation, linuxCRC32);
-      if (fs.existsSync(candidatePath)) {
-        returnValue = candidatePath;
-      }
-    }
-  } else if (process.platform == "win32") {
-    if (hasbin.sync(win32CRC32)) {
-      returnValue = win32CRC32;
-    } else if (installationLocation.length > 0) {
-      let candidatePath = path.join(
-        installationLocation,
-        exeFilename(win32CRC32)
-      );
+  if (vcastInstallationPath) {
+      let candidatePath = path.join(vcastInstallationPath, crc32Name);
       if (fs.existsSync(candidatePath)) {
+        vectorMessage(`   found '${crc32Name}' here: ${vcastInstallationPath}`);
         returnValue = candidatePath;
       }
```

Comment:
Should we show a `VectorMessage` here if `candidatePath` does not exist, consistent with how we handle this for other executables?

Reply:
This is special because it is not fatal and we put up a pop up message to the user at 
C:\RDS\vector-vscode-vcast\src\utilities.ts:486

Reply:
Ok, I added a message to the log!
test explorer  [info]     could NOT find 'crc32-win32.exe' here: c:\vcast\vc22sp7, coverage annotations will not be available